app/models/export/activity_forecast_columns.rb

```diff
@@ -1,7 +1,12 @@
 class Export::ActivityForecastColumns
-  def initialize(activities:, report: nil)
+  def initialize(activities:, report: nil, starting_financial_quarter: nil)
     @activities = activities
     @report = report
+    @starting_financial_quarter = starting_financial_quarter
+
+    if @report && @starting_financial_quarter
+      raise ArgumentError.new("Only use report OR starting_financial_quarter")
```

Comment:
could we be a little more helpful with this error message, e.g. something like:

```
"Provide either a report for historical forecasts or a starting_financial_quarter for the latest forecasts"
```

?